keras/src/initializers/__init__.py

```diff
@@ -110,6 +133,22 @@ def get(identifier):
     elif isinstance(identifier, str):
         config = {"class_name": str(identifier), "config": {}}
         obj = deserialize(config)
+    elif ops.is_tensor(identifier) or isinstance(
+        identifier, (np.generic, np.ndarray)
+    ):
+
+        def initialize_fn(shape, dtype=None):
```

Comment:
I just realized -- this approach won't work because it can't be serialized correctly. If you do `Dense(... kernel_initializer=value)`, to keep the layer serializable, the initializer must be serializable too.

We already have a serializable initializer that handles constant values: `Constant`

So you can just do:

```
elif ops.is_tensor(identifier) or isinstance(
        identifier, (np.generic, np.ndarray)
):
    return Constant(identifier)
```

or something along those lines

Reply:
Yes, this type of initializer cannot be serialized (similar to passing a callable function).

Using `Constant` for a large tensor is unsuitable because the memory cannot be released. This is why we use a lambda expression in `quantize`.

I think using `initializer.get` with a callable fn or tensor should be for internal use within a layer or model. I can add docstrings for this.

Reply:
Ok, got it